commerce_coordinator/settings/local.py

```diff
@@ -113,7 +113,8 @@
 
 ENABLE_AUTO_AUTH = True
 
-LOGGING = get_logger_config(debug=DEBUG)
+LOGGING_FORMAT_STRING = os.environ.get("LOGGING_FORMAT_STRING", "")
```

Comment:
This will always result in an empty string as we have not set its value for local setup anywhere. So, no need to get its value and pass it to the logger function as it already defaults to an empty string, i.e., `format_string=''`

Reply:
In this way, we can set its value in private.py. No?

Reply:
this change has been reverted. now we rely on `DEFAULT_LOGGING_FORMAT_STRING`